backend/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const app = express();
+const orm = require("sequelize");
+const database = new orm.Sequelize({
+  dialect: "sqlite",
+  storage: "./data/catsDatabase.sqlite",
+});
+
+const Cat = database.define("Cat", {
+  name: {
+    type: orm.DataTypes.STRING,
+    allowNull: false,
+  },
+  imageUrl: {
+    type: orm.DataTypes.STRING,
+    allowNull: false,
+  },
+  age: {
+    type: orm.DataTypes.INTEGER,
+    allowNull: false,
+  },
+  description: {
+    type: orm.DataTypes.STRING,
+  },
+  available: {
+    type: orm.DataTypes.BOOLEAN,
+    defaultValue: true,
+  },
+});
+
+async function createCatsHandler(req, res) {
+  if (

```

Comment:
Use a validator or schema based validation

 (EXAMPLE: https://www.npmjs.com/package/class-validator/v/0.6.0)

Reply:
If not validator
Prefer a separe function to validate each prop

like 

```javascript
function validateCreateCatPayload(payload) {} // return boolean
```